acinclude/squid-util.m4

```diff
@@ -214,6 +214,17 @@ AC_DEFUN([SQUID_YESNO],[
   AS_IF([test "$1" != "yes" -a "$1" != "no"],[AC_MSG_ERROR([Bad argument for $2: "$1". Expecting "yes", "no", or no argument.])])
 ])
 
+dnl Check that a library is actually available, useable,
+dnl and where its pieces are (eg headers and hack macros)
+dnl Parameters for this macro are:
+dnl 1) binary library name (without 'lib' prefix)
```

Comment:
The word "binary" looks out of place in this phrase IMHO. It is probably unnecessary.

```suggestion
dnl 1) library name (without 'lib' prefix)
```